coniferest/evaluator.py

```diff
@@ -107,6 +107,14 @@ def score_samples(self, x):
                 / (self.average_path_length(self.samples) * trees)
         )
 
+    def pariou_transform(self, x, axis = None):
```

Comment:
We need a better name! `feature_signature`? `feature_score`?

Reply:
We renamed it to `feature_signature` and `feature_importance`.